images/tester/tests/test_dag_consistency.py

```diff
@@ -2,15 +2,15 @@
 
 import pytest
 
-from tests import DAG_PATH, DAG_FILES
+from tests import DAG_FILES, DAG_PATH
 
 
-@pytest.mark.parametrize('dag_file', DAG_FILES)
+@pytest.mark.parametrize("dag_file", DAG_FILES)
 def test_all_dags_in_dag_bag(dag_bag, dag_file):
     """Assert that all the dags for which there are files exist in the airflow dag bag.
     """
     dag_files = {
-        os.path.split(dag.full_filepath)[-1]
+        os.path.join(*dag.full_filepath.split(os.path.sep)[3:])
```

Comment:
What's the significance of `[3:]`

Reply:
`os.path.split('/airflow/dags/foo') == ['', 'airflow', 'dags', 'foo']`